### PR TITLE
Backend 32 sensor endpoint

### PR DESCRIPTION
**Pull request by MystikHub**
Sensor data can be received while running, but some issues appear in tests:

- /vote test returns a code 500 response
- /sensorSubmit test returns a timeout exception

### CODE REVIEW

File: backend/sensorStore.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    populate(){
+        let rawdata = fs.readFileSync('mockSensorData.json');
+        let sensors = JSON.parse(rawdata).sensors;
+        this.keys = sensors
+    }
+    
+    store(id, location, temperature, time) {
+        if (isFinite(id)) {     // had issues with "instanceof"
```
Comment (henrym2) [inline backend/sensorStore.js]: I think checking keys by ID wont work, should probably do a for loop to iterate through and find the associated item.

File: backend/request [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "time": "01:02:03T20:12:2020"
+}
+
+# POST http://localhost:3000/vote HTTP/1.1
+# content-type: application/json
+
+# {
+#     "submitter": "John",
+#     "opinion": -1
+# }
```
Comment (henrym2) [inline backend/request]: What is this for?

Comment (MystikHub) [reply backend/request]: That's for the REST extension for VS Code. If you press CTRL + ALT + R in that file, it'll send a HTTP request with the specified data

**Comment by henrym2** [comment]
Huh okay cool I'll keep that in mind, right I'll approve this and we can
merge it in

**Pull request merged**